locale/en/LC_MESSAGES/django.po

```diff
@@ -501,6 +525,26 @@ msgid ""
 "change, please contact the administrator."
 msgstr ""
 
+#: templates/templated_email/source/export_failed.mjml:16
+msgctxt "Export products e-mail text"
+msgid ""
+"\n"
+"            Sorry, we couldn't finish exporting products, some unexpected "
+"errors occurred.\n"
+"            Please try again.\n"
+"        "
```

Comment:
We shouldn't have newline characters and empty spaces here, because it's harder to provide translations then. All `blocktrans` tags should also have the `trimmed` parameter. See what [the docs](https://docs.djangoproject.com/en/3.0/topics/i18n/translation/) say about it and how we use blocktrans in other email templates.